native/package.json

```diff
@@ -41,7 +41,8 @@
     "lint:changed": "eslint . -c ../.eslintrc_changed.js",
     "lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml",
     "eslint": "eslint .",
-    "postinstall": "jetifier && patch-package",
+    "license-crawler": "npm-license-crawler -onlyDirectDependencies -json ./assets/licenses.json",
```

Comment:
```suggestion
    "license-crawler": "npm-license-crawler -onlyDirectDependencies -json ./assets/licenses_native.json"
```

Reply:
since we create a license file for web & native, you could also keep the naming "licenses" and just rename it in the import statement

Reply:
@f1sh1918 so we really have one common license file? I thought that only the native licenses get included into this file

Reply:
Fine as it is, since it now lays in `native/assets`, the name `licenses.json` is enough.